routing/router.go

```diff
@@ -288,6 +289,10 @@ type Config struct {
 	// we need in order to properly maintain the channel graph.
 	ChainView chainview.FilteredChainView
 
+	// Notifier is a reference to the ChainNotifier, used to grab
```

Comment:
Commit body could use some additional context w.r.t the motivation for this change, what went wrong w/ things before, etc, etc.